Install_or_Update.bat

```diff
@@ -1,8 +1,14 @@
-set PATH=%PATH%;%USERPROFILE%\anaconda3\Scripts;
-call activate.bat
-call conda remove --name geoapps --all
-call conda env update --file environment.yml  --prune
-call activate.bat geoapps
-call conda env config vars set KMP_WARNINGS=0
+@echo off
+setlocal EnableDelayedExpansion
+
+call %~dp0get_conda_exec.bat
+if !errorlevel! neq 0 (
+  exit /B !errorlevel!
+)
+
+cd %~dp0
+call !MY_CONDA_EXE! remove --name geoapps --all --yes
+call !MY_CONDA_EXE! env update --file environment.yml --prune
+call !MY_CONDA_EXE! activate geoapps && python -m pip install -e . --no-deps
 pause
```

Comment:
Ok not sure what I am doing wrong here, but I am just getting this when running the bat file
![image](https://user-images.githubusercontent.com/55204635/138357395-a78c5f1e-58be-4d91-979d-652bdb4c8ecc.png)

No installation triggered?

Reply:
but it did not fail, you would not have the "Using Conda" message

Reply:
yea I dont know man. Still not working here.